### ISSUE TITLE
copycat+ blocks not showing on litematics 1.20.1

### ISSUE THREAD

**Issue by Kieran135642** [maruohon/litematica#897]
When trying to use litematica with copycat+ blocks they dont seem to show visually on the litematic although they do show in the material list also if I know where they are in their placement and i have them in my inventory i can use middle mouse button and it goes brings it to my hotbar so not sure why its not visually there. Im pretty sure ive seen them work in earlier versions of the game.
(Not sure if this is a problem with litematica or copycat+)
Game version 1.20.1
Mod version 1.20.1-0.15.3

**Comment by maruohon** [comment]
I'm assuming they must use the Fabric rendering API for those blocks in some way then, maybe? Litematica doesn't currently use/do anything with that API. Hopefully after the rendering rewrites in Litematica I'll get around to looking into the rendering API and how it works and what support it needs from the mod.

**Comment by Kieran135642** [comment]
Yeah, I do have Fabric API in my mod list so i guess that must be the case.

**Comment by Kieran135642** [comment]
Yeah, I do have Fabric API in my mod list so i guess that must be the case.

**Comment by andreholman** [comment]
Adding on that you can work your way around this by using the schematic verifier and selecting the copycat block types that you want to identify. This is not feasible large-scale, but better than guessing.

**Comment by Kieran135642** [comment]
Unfortunately it was quite a big build and so that wasnt possible, luckily playing modded minecraft I was able to use create mod to build it instead with its schematic cannon
